src/views/GovernanceReviewTeam/index.tsx

```diff
@@ -158,13 +159,26 @@ const GovernanceReviewTeam = () => {
               </li>
             </ul>
             <hr />
-            <Link
-              to={`/governance-review-team/${systemId}/actions`}
-              aria-label={t('actions')}
-              className={getNavLinkClasses('actions')}
-            >
-              {t('actions')}
-            </Link>
+            <ul className="easi-grt__nav-list">
+              <li>
+                <Link
+                  to={`/governance-review-team/${systemId}/actions`}
+                  aria-label={t('actions')}
```

Comment:
You don't need this `arial-label` if it's the same as the text that's rendered in the link. You'd only need the `aria-label` if you wanted to render text with more context than what's displayed.

Same goes for the notes link too.